index.html

```diff
@@ -0,0 +1,146 @@
+                    after a few weekends running on Sunday, I noticed all the enjoyable time that 
+                    I spend while running, so, here is when the storey begins. I enrolled in my 
+                    first race, and after in the second and continue until I saw that I want more,
+                    more challenges for me. I started the race to earn the Abbot Marathon Major Medal,
+                    only people who run the six majors can earn it. I'm writing this page expecting to
+                    inspire someone to start to run.
+                </p>
+            </div>
+            <h2>Raise to the six star marathon major medal</h2>
+            <p class="p-world">At all around the world!</p>
+        </div>
+        <div class="marathon-section">
+            <h3 class="subtitles">Six Star Marathon Majors</h3>
+            <div class="d-flex marathon-majors">
+                <div class="marathon-info d-flex">
+                    <img src="./images/berlinIcon.svg" alt="Berlin marathon logo">
+                    <p class="city">Berlin</p>
+                    <p class="slogan">Race to the gate...</p>
+                </div>
+                <div class="marathon-info d-flex">
+                    <img src="./images/newyorkIcon.png" alt="New York marathon logo">
+                    <p class="city">New York</p>
+                    <p class="slogan">Rule the five boroughs...</p>
+                </div>
+                <div class="marathon-info d-flex">
+                    <img src="./images/tokyoIcon.png" alt="Tokyo marathon logo">
+                    <p class="city">Tokyo</p>
+                    <p class="slogan">Explore the old and the new...</p>
+                </div>
+                <div class="marathon-info d-flex">
+                    <img src="./images/chicagoIcon.png" alt="Chicago marathon logo">
+                    <p class="city">Chicago</p>
+                    <p class="slogan">Tame the windy city...</p>
+                </div>
+                <div class="marathon-info d-flex">
+                    <img src="./images/londonIcon.png" alt="London marathon logo">
+                    <p class="city">London</p>
+                    <p class="slogan">Royal park to royal palace...</p>
+                </div>
+                <div class="marathon-info d-flex">
+                    <img src="./images/bostonIcon.png" alt="Boston marathon logo">
+                    <p class="city">Boston</p>
+                    <p class="slogan">Conquer the original...</p>
+                </div>
+                <button class="marathon-button primary-btn">Berlin's Journey</button>
+            </div>
+            <a href="#" class="see-more"><p>MORE MARATHONS IN THE WORLD</p></a>
+        </div>
+        <div class="highlights-section" id="races">
+            <h3 class="subtitles">Highlight Races</h3>
+            <div class="races d-flex"></div>
```

Comment:
- [ ] It would be better if the images on Highlight Races were the same size and align properly as the original design. 

###original design

![Capture](https://user-images.githubusercontent.com/64173189/137533425-2698d6ce-4a28-4feb-a5c5-ea79a611a9d6.JPG)

###Your design

![Capture1](https://user-images.githubusercontent.com/64173189/137533448-d9c7c5fb-631c-4d31-b6f5-b1d7a603006c.JPG)